package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "json-masker",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
To me, the minor version should be rather incremented. This is a new feature, not a bug fix.